source/contextHelp.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: UTF-8 -*-
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2017-2018 NV Access Limited, Thomas Stivers
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+import gui
+import ui
+import wx
+from logHandler import log
+from xml.etree import ElementTree
+
+class Help():

```

Comment:
This class doesn't seem to be used anywhere. Note that class names usually start with an upper case letter and inherrit from object.